doc/user/command-line.xml

```diff
@@ -1639,6 +1654,29 @@ foo.c
           <scons_output_command>scons -Q COLORS=magenta foo.o</scons_output_command>
         </scons_output>
 
+        <para>
+
+        You can use this last characteristic as a way to enforce at least
+        one of your valid options being chosen by specifying the valid
+        values with the <parameter>names</parameter> parameter and then
+        giving a value not in that list as the <parameter>default</parameter>
+        parameter - that way if no value is given on the command line,
+        the default is chosen, and it will error out with an invalid value.
+        The example is, in fact, set up that way by using
+        <literal>0</literal> as the default:
```

Comment:
can you add an example?

Reply:
there is an example, though you can't read it inline - it's the newly generated  doc/generated/examples/commandline_ListVariable_4.xml elsehwere in this patch.